expression/builtin_miscellaneous_vec.go

```diff
@@ -57,3 +57,29 @@ func (b *builtinInetNtoaSig) vecEvalString(input *chunk.Chunk, result *chunk.Col
 func (b *builtinInetNtoaSig) vectorized() bool {
 	return true
 }
+
+func (b *builtinIsIPv4Sig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+	result.ResizeInt64(n, false)
+	i64s := result.Int64s()
```

Comment:
we should calculate null bitmap for `result` according to the input parameter `buf`.

Reply:
If I understand correctly, when the input row is "", `result`'s null bitmap for that row should be `true`?

However, looking at the current implementation of `builtinIsIPv4Sig`, it returns `true` only when `err != nil`, but not when the input string `isNull` is `true`. i.e. `builtinIsIPv4Sig.evalInt()` 2nd return value is `false` even if the input string is null. 

Anything I'm missing here? Appreciate further clarification!

Reply:
```
TiDB(root@127.0.0.1:test) > select is_ipv4(null);
+---------------+
| is_ipv4(null) |
+---------------+
|             0 |
+---------------+
1 row in set (0.01 sec)
```

According to the current behavior, when the input is null, we should return false. How about adding a comment about this behavior to eliminate the confusion?

Reply:
Sounds good, I added a comment.